gerrit_to_github_issues/engine.py

```diff
@@ -15,23 +15,29 @@
 import github
 import pytz as pytz
 from github.Repository import Repository
+from github.Project import Project
+from github.Issue import Issue
 
 from gerrit_to_github_issues import gerrit
 from gerrit_to_github_issues import github_issues
 
 LOG = logging.getLogger(__name__)
 
 
-def update(gerrit_url: str, gerrit_project_name: str, github_project_name: str, github_user: str, github_password: str,
-           github_token: str, change_age: str = None, skip_approvals: bool = False):
-    gh, repo = github_issues.get_repo(github_project_name, github_user, github_password, github_token)
+def update(gerrit_url: str, gerrit_project_name: str, github_project_id: int,
```

Comment:
doesn't look like `gerrit_project_name` is getting passed in anymore, since it was removed from cli.py parser arguments

Reply:
good catch - `gerrit_project_name` has been renamed to `gerrit_repo_name`